internal/services/appservice/helpers/shared_schema.go

```diff
@@ -1157,6 +1171,12 @@ func ExpandIpRestrictions(restrictions []IpRestriction) (*[]webapps.IPSecurityRe
 
 		restriction.Priority = pointer.To(v.Priority)
 
+		if v.Description != "" {
+			restriction.Description = utils.String(v.Description)
+		}
+
+		restriction.Priority = utils.Int64(v.Priority)
```

Comment:
Can we replace these with `pointer.To()` as we're aiming to remove the `utils` package?
```suggestion
			restriction.Description = pointer.To(v.Description)
		}

		restriction.Priority = pointer.To(v.Priority)
```